packages/server/graphql/intranetSchema/queries/company.ts

```diff
@@ -17,10 +18,14 @@ const company = {
     }
   },
   description: 'All the info about a specific company',
-  async resolve(_source: unknown, {domain, userId}, {authToken, dataLoader}) {
+  async resolve(
+    _source: unknown,
+    {domain, userId}: {domain?: string; userId?: string},
```

Comment:
+1 optional GraphQL parameters should be `domain: string | null | undefined` since `null` will be passed when omitted